ddtrace/contrib/aiobotocore/patch.py

```diff
@@ -147,7 +147,7 @@ async def _wrapped_api_call(original_func, instance, args, kwargs):
         meta = {
             "aws.agent": "aiobotocore",
             "aws.operation": operation,
-            "aws.region": region_name,
+            "region": region_name,
```

Comment:
If I am editing ddtrace/ext/aws.py then aiobotocore should also be on par

Reply:
additionally: In order to achieve pivoting from metrics to traces to work out of the box we need these new tags. For instance if we wish to pivot from aws integration metric `aws.dynamodb.table_size` then our spans metadata needs to match with metrics metadata. This also allows us to unlock benefit of inferring related resources on the serverless function page.

<img width="1007" alt="Screen Shot 2023-04-27 at 8 07 07 PM" src="https://user-images.githubusercontent.com/49878080/235045306-5bfc816a-d437-472c-b901-c8e5abe41564.png">


Why not remove old tags such as aws.region, or aws.sqs.queue_name now? Many of these tags are already used by customer monitors, dashboards, metrics, notebooks... Therefore I have logged a jira ticket under framework integrations to eventually phase these old tags out.

If the aws integrations metrics could change their metric tag values then this would solve the pivot issue but changing a tag on a metric like that is a much bigger breaking change as much more customer resources rely on those.